mix.exs

```diff
@@ -71,7 +75,7 @@ defmodule Authoritex.MixProject do
 
   defp package do
     [
-      files: ["lib", "mix.exs", "README.md", "LICENSE.md"],
+      files: ["lib", "mix.exs", "README.md", "LICENSE.md", "AUTHORITEX.livemd"],
```

Comment:
Should this be another reference to `LIVEBOOK.livemd`?

Reply:
Thanks for catching that!